platform/platform_mac.mm

```diff
@@ -62,10 +62,7 @@
     std::string const paths[] =
     {
       // Developers can set a symlink to the data folder.
-      m_resourcesDir + "../../../data/",
-      // Check development environment without a symlink but with a git repo.
-      m_resourcesDir + "../../../../omim/data/",
-      m_resourcesDir + "../../../../organicmaps/data/",
+      m_resourcesDir + "../../../../data/",
       // Working directory is set to the data folder or any project's subfolder.
       currentDir + "/../data",
```

Comment:
Maybe just pass `OMIM_DATA_DIR` from cmake?
https://github.com/organicmaps/organicmaps/blob/e3f0e43d9be711d55d201f0f533fbd256e185dca/cmake/OmimConfig.cmake#L7-L8

Reply:
No absolute paths please.

Reply:
I don't understand the point of those "symlinks"
They shouldn't be present on release builds at all
If we want our app to use the data from repo_root/data dir then the pass to this dir should be explicitly specified

On release builds it should either be configurable via app (like set workspace path window at first start or during the install process) or system-dependent paths should be specified:
* MacOS - App.dir/Contents/data/
* Windows - C:/ProgramFiles/...
* Linux - only god knows which path to use...

This list of "symlinks" is the weirdest solution I've ever seen